sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -129,13 +129,14 @@ namespace Azure { namespace Security { namespace Attestation {
      * @param credential The authentication method to use (required for TPM attestation). If the
      * credential parameter is not supplied, the connection will be unauthenticated.
      * @param options The options to customize the client behavior.
+     * @param context The context for cancelling long running operations.
      * @return The newly created client.
      */
     static AttestationClient Create(
         std::string const& endpoint,
         std::shared_ptr<Core::Credentials::TokenCredential const> credential,
         AttestationClientOptions const& options = AttestationClientOptions{},
-        Azure::Core::Context const& constext = Azure::Core::Context{});
+        Azure::Core::Context const& context = Azure::Core::Context{});
```

Comment:
I believe this is generally considered a non-breaking change. C++ (14 and lower) doesn't have named parameters, correct?

Reply:
That is correct - the C++ language doesn't have named parameters, unlike C#.